### PR TITLE
Feat/flex layout

### PR DESCRIPTION
**Pull request by ayelenjaramillo**
added code with flexbox and some media queries (tablet and desktop)

### CODE REVIEW

**Comment by ezequiel-baruf-lenio** [comment]
@ayelenjaramillo are issue a PR to `setup-proyect` branch for some reason?
Makes sense if you are going to merge some branches into `setup-proyect` and then that one to master.

**Comment by ayelenjaramillo** [comment]
> @ayelenjaramillo are issue a PR to `setup-proyect` branch for some reason? Makes sense if you are going to merge some branches into `setup-proyect` and then that one to master.

I have already changed the branch.

**Comment by GomezAgustin1** [comment]
Good Work

**Review by GomezAgustin1 — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <div class="right">
+      <img src="./assets/Line.png" class="img"/>
+      <img src="./assets/🦆 icon _search_.svg" class="img"/>
+      <img src="./assets/random_girl.png" alt="random_girl" class="random_g"/>
+    </div>
+ </nav>
+</header>
+<body>
+  <div class="general_container">
+    <span>
```
[the remark this answers is not in the document]
Comment (ayelenjaramillo) [reply index.html]: thanks for the recommendation. I´ll keep it in mind the next time!

**Pull request merged**